extensions-core/druid-pac4j/src/main/java/org/apache/druid/security/pac4j/OIDCConfig.java

```diff
@@ -26,6 +26,7 @@
 
 public class OIDCConfig
 {
+  private final String DEFAULT_SCOPE = "name";
```

Comment:
The default claim to use for the identity is `sub` since `name` is not a standard OIDC claim

Reply:
`name` is a standard claim according to the [OIDC spec](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims).

Reply:
While it is a standard claim, it isn't mandatory.  Some OIDC compliant auth servers don't include that claim in the ID token even when the `profile` scope is requested.